kubeflow/gcp/prototypes/iap-ingress.jsonnet

```diff
@@ -14,6 +14,7 @@
 // @optionalParam privateGKECluster string false Is the k8s cluster a private GKE cluster
 // @optionalParam useIstio string false The namespace where Istio is installed
 // @optionalParam istioNamespace string istio-system The namespace where Istio is installed
+// @optionalParam  espSampleAppImage string gcr.io/cloud-solutions-group/esp-sample-app:1.0.0 The sample app used with IAP
```

Comment:
extra space after @optionalParam

Reply:
Done thanks; lets hope that fixed the test.